index.js

```diff
@@ -1,12 +1,12 @@
 'use strict';
 
-var React = require('react-native');
+import React, { Component } from 'react';
 
-var {
-  View,
-  StyleSheet,
-  requireNativeComponent,
-} = React;
+import {
+    View,
```

Comment:
Thanks for the PR! Can you use 2 spaces for indentation here please and I'll merge it in.